tests/test_unit/test_sample_data.py

```diff
@@ -127,28 +134,79 @@ def test_list_datasets(valid_sample_datasets):
     assert all(file in list_datasets() for file in valid_sample_datasets)
```

Comment:
This needs to be updated if we decide to parametrise the `sample_dataset` fixture
```suggestion
def test_list_datasets():
    assert isinstance(list_datasets(), list)
    assert all(file in list_datasets() for file in SAMPLE_DATASETS)
```